src/components/DownloadManifestModal.js

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import { compose, withState } from 'recompose';
+import { injectState } from 'freactal/lib/inject';
+import { css } from 'emotion';
+import { withTheme } from 'emotion-theming';
+import { Trans } from 'react-i18next';
+
+import downloadIcon from '../assets/icon-download-white.svg';
+import IconWithLoading from '../icons/IconWithLoading';
+import CopyToClipboard from './CopyToClipboard';
+import { ModalFooter, ModalWarning } from './Modal';
+import { FileRepoStats } from './Stats';
+import LoadingOnClick from 'components/LoadingOnClick';
+import graphql from '../services/arranger';
+import Spinner from 'react-spinkit';
+
+const Button = compose(withTheme)(({ theme, children, ...props }) => (
+  <button css={theme.actionButton} {...props}>
+    <div
+      className={css`
+        display: flex;
+        justify-content: center;
+        align-items: center;
+      `}
+    >
+      {children}
+    </div>
+  </button>
+));
+
+export const SubHeader = ({ children, ...props }) => (
+  <div
+    className={css`
+      margin-bottom: 9px;
+      font-size: 15px;
+      font-weight: 600;
+      line-height: 1.87;
+      letter-spacing: 0.2px;
+      color: #343434;
+    `}
```

Comment:
This looks like some pretty stylistic css, I'm thinking we should try to add class names to these so Kim can address them more easily if needed :)